src/scripts/content/dokuwiki.js

```diff
@@ -0,0 +1,18 @@
+/*jslint indent: 2, unparam: true*/
+/*global $: false, document: false, togglbutton: false*/
+
+'use strict';
+togglbutton.render('#dokuwiki__content', {observe: false}, function (elem) {
+  var link, description,
+    numElem = $(".pageId span"),
+    pName = numElem.textContent.split(":")[0].trim();
+
+  description = numElem.textContent.split(" ").pop().trim();
+  link = togglbutton.createTimerLink({
+    className: 'wiki',
+    description: description,
+    projectName: pName
+  });
+
+  $(".wrapper.group, .pageId").prepend(link);
```

Comment:
I'd do something like:

```javascript
var target = $(".wrapper.group") // DokuWiki Template
           || $(".pageId");      // Bootstrap3 Template

target.prepend(link);
```

To  pin-point which selector is effective of which template.